archive/frames/utils.py

```diff
@@ -16,8 +16,8 @@
 
 @lru_cache(maxsize=1)
 def get_s3_client():
-    config = boto3.session.Config(region_name='us-west-2', signature_version='s3v4')
```

Comment:
whats the effect of removing the region_name? Is it specified elsewhere or defaults or something, or is it just not needed anymore? We probably shouldn't have a region name hardcoded anyway though since this is supposed to be general

Reply:
The region name automatically uses the AWS_DEFAULT_REGION environment variable if it exists (just like the AWS_ACCESS_KEY_ID and AWS_SECRET_ACCESS_KEY). 

I could load the region in settings, and add `region_name=settings.AWS_DEFAULT_REGION` if we want to be more visually explicit about it.